src/test/shell/bazel/bazel_workspace_status_test.sh

```diff
@@ -48,7 +48,7 @@ EOF
 function test_workspace_status_parameters() {
   create_new_workspace
 
-  local cmd=$TEST_TMPDIR/status.sh
+  local cmd=`mktemp $TEST_TMPDIR/wsc-XXXXXXXX.sh`
```

Comment:
Please remove the `.sh` ending here and in other `mktemp` calls.
On macOS `mktemp` requires that the XXXs be at the end of the template, otherwise the command creates a file called literally `wsc-XXXXXXXX.sh`.

Reply:
Good point - thank you!  Fixed.